File: my-folder/0557-reverse-words-in-a-string-iii/solution.java
class Solution {
    public String reverseWords(String s) {
        String[] words = s.split("\\s+");
        StringBuilder res = new StringBuilder();
        for (String word : words) {
            StringBuilder rev = new StringBuilder(word);
            rev.reverse();
            res.append(rev).append(" ");
        }
        return res.toString().trim();
    }
}

File: my-folder/0557-reverse-words-in-a-string-iii/solution.py
class Solution:
    def reverseWords(self, s: str) -> str:
        arr = s.split(" ")
        reversed_words = []
        for word in arr:
            reversed_word = word[::-1]
            reversed_words.append(reversed_word)
        res = " ".join(reversed_words)
        return res
